ocs_ci/ocs/resources/ocs.py

```diff
@@ -224,7 +224,17 @@ def get_ocs_csv():
         subscription_plan_approval=subscription_plan_approval,
     )
     channel = config.DEPLOYMENT.get("ocs_csv_channel")
-    ocs_csv_name = ocs_package_manifest.get_current_csv(channel=channel)
```

Comment:
Can you explain why you moved ocs_csv_name under the else statement?
Because for a scenario where we  hit the `if` conditional, and we do not hit the conditional `item["metadata"]["name"].startswith(defaults.OCS_OPERATOR_NAME):` we could end up with `ocs_csv_name` not being assigned, and this line will receive an error:
`ocs_csv = CSV(resource_name=ocs_csv_name, namespace=namespace)`

Reply:
If the ocs_csv_name is not assigned, means the OCS csv is not available and have made the changes to raise for the same. Moving `ocs_csv_name` under else to make sure it only executes for non openshift dedicated platforms. In case of openshift dedicated the OCS CSV is pulled from available CSVs instead of packagemanifests.

Reply:
Thanks :)